.github/workflows/build-docs.yaml

```
@@ -0,0 +1,23 @@
+---
+name: build documentation
+
+on:
+  pull_request:
+    branches:
+    - main
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with:
+          python-version: 3.x
+      - name: Install Poetry
+        run: |
+          pip install poetry
+      - name: Install Dependencies
+        run: |
+          poetry install
```

Once my [suggestion applied](https://github.com/scality/gh-actions-exporter/pull/18/files#r1174026094)
```suggestion
      - name: Install Dependencies
        run: |
          poetry install --only docs
```